ee/management/commands/migrate_clickhouse.py

```diff
@@ -71,18 +79,35 @@ def migrate(self, host, options):
                 )
             print("Migrations done")
         else:
+            is_fresh_instance = len(self.get_applied_migrations(database)) == 0
             database.migrate(MIGRATIONS_PACKAGE_NAME, options["upto"], replicated=CLICKHOUSE_REPLICATION)
-            print("Migration successful")
+            if is_fresh_instance and not options["skip_special_migrations"]:
+                for migration_name, definition in ALL_SPECIAL_MIGRATIONS.items():
+                    sm = SpecialMigration.objects.get_or_create(
+                        name=migration_name,
+                        description=definition.description,
+                        posthog_min_version=definition.posthog_min_version,
+                        posthog_max_version=definition.posthog_max_version,
+                    )[0]
+                    if sm.status == MigrationStatus.NotStarted:
+                        print("Applying special migration", migration_name)
```

Comment:
can we use the structured logs we have in the app instead of `print`? Example:

```
>>> import structlog
>>> logger = structlog.get_logger(__name__)
>>> logger.info("an_error_occurred", bar="Buz")
```